src/main/webapp/teacher/controlGameInstance/controlGameInstance.js

```diff
@@ -235,13 +237,63 @@ function initGameInstanceListener(gameInstanceId) {
     // The listener to the question stats is initiated
     initQuestionStatsListener({ gameInstanceId, currentQuestionId: gameInstanceUpdate.currentQuestion });
 
+    initQuestionAnswerStatsListener(gameInstanceId, gameInstanceUpdate.currentQuestion);
+
     updateNumberOfMembersUI(gameInstanceUpdate.numberOfMembers);
 
   });
 }
 
-// Updates the panel showing which questions students are seeing
+function initQuestionAnswerStatsListener(gameInstanceId, currentQuestionId) {
+
+  if (unsubscribeCurrentActiveQuestionAnswersInGameInstance != null) {
+    unsubscribeCurrentActiveQuestionAnswersInGameInstance();
+  }
+  unsubscribeCurrentActiveQuestionAnswersInGameInstance = db.collection('gameInstance').doc(gameInstanceId).collection('questions').doc(currentQuestionId).collection('answers')
```

Comment:
I think you don't need two listeners? If answers is a subcollection on the same currentQuestion document you can just listen to that document, have a global variable for the number of answers (that you retrieve when the page loads), and then just make a single query to check if it's different from the one you have stored, if it is then you updateCurrentQuestionAnswerStats(). I think this would be more efficient than having two listeners